docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -19,32 +15,167 @@ We'll do our best to keep things easy to follow, and as sequential as possible,
 This documentation page covers the _how_ of migrating from v4 to v5.
 The _why_ will be covered in an upcoming blog post on Medium.
 
-## Updating your dependencies
+## Migration Steps
+
+- [Update React & TypeScript](#update-react-&-typescript-version)
+- [ThemeProvider Setup](#themeprovider-setup)
+- [Update Material-UI](#update-material-ui-version)
+- [Run Codemod](#run-codemod)
+  - [🪄 preset-safe](#preset-safe)
```

Comment:
```suggestion
  - [🪄 preset-safe](#🪄preset-safe)
```

Reply:
Maybe 
```suggestion
  - [preset-safe](#preset-safe)
```

I see this on both the link and the header
![image](https://user-images.githubusercontent.com/4512430/125440589-926f2c85-0b69-4856-ab06-4198807cfd56.png)

Reply:
okay, will remove the emoji.